rules/aws_vpc_flow_rules/aws_dns_crypto_domain.yml

```diff
@@ -0,0 +1,141 @@
+AnalysisType: rule
```

Comment:
Should we add a dedup threshold or timeframe? I think it could be possible an alert storm is created depending on how often the crypto miner checks DNS.

Reply:
resolved offline